tcpdf.php

```diff
@@ -6409,7 +6409,7 @@ public function Write($h, $txt, $link='', $fill=false, $align='', $ln=false, $st
 		// calculate maximum width for a single character on string
 		$chrw = $this->GetArrStringWidth($chars, '', '', 0, true);
 		array_walk($chrw, array($this, 'getRawCharWidth'));
-		$maxchwidth = max($chrw);
+		$maxchwidth = ((is_array($chrw) || $chrw instanceof Countable) && count($chrw) > 0) ? max($chrw) : 0;
```

Comment:
If `$chrw` is not an `array`, then the above `array_walk()` would also cause a warning. So it seems this should be something along the lines of:

```php
$maxchwidth = 0;

if (is_array($chrw) && count($chrw) > 0) {
    array_walk($chrw, array($this, 'getRawCharWidth'));
    $maxchwidth = max($chrw);
}
```

I did not see any places where `GetArrStringWidth()` could return `object`, so I omitted the `instanceof` check